baseline.yaml

```diff
@@ -720,6 +720,62 @@ criteria:
       in a separate repository and fetched during
       a specific well-documented pipeline step.
     control_mappings: # TODO
+    security_insights_value: # TODO
+    scorecard_probe: # TODO
+  - id: OSPS-53
+    maturity_level: 2
+    category: Build & Release
+    criteria: |
+      All released software assets MUST be signed or accounted for in a signed manifest including each asset's cryptographic hashes (such as an SBOM, provenance attestation or similar). The identity of the person or process authoring the software release must be published, discoverable and must be taken into account during the verification process.
+      with a cryptographic signature.    
+    objective: |
+      Provide users with a mechanism to verify the
+      authenticity and integrity of released 
+      software assets, reducing the risk of
+      tampering or unauthorized modifications.  
+    implementation: |
+      Sign all released software assets at build 
+      time with a cryptographic signature or attestations, such 
+      as GPG or PGP signature, Sigstore signatures, SLSA provenance, or SLSA VSAs.   
+      
+      Provide clear documentation for users on how they can verify the signature
+      and what a valid signature or attestation indicates (e.g.
+      "it came from our CI/CD system",
+      "one of our project maintainers built the software", "it was built from the
+      indicated source repo with the indicated builder", "it meets SLSA level 3").
+      
+      It is recommended that this signature is
+      generated as part of the build and release
+      pipeline to ensure that it is consistent and
+      automated.
+    control_mappings: # TODO
+    security_insights_value: # TODO
+    scorecard_probe:
+      - releasesAreSigned 
+  - id: OSPS-54
```

Comment:
I'd suggest splitting this out into a separate PR, since it seems less controversial than the other one.